packages/@aws-cdk/custom-resources/lib/aws-custom-resource/sdk-api-metadata.json

```diff
@@ -0,0 +1,771 @@
+{
```

Comment:
This was previously gitignored => update `.gitignore` and `cdk-build.pre` key in `package.json` with new location?